oper8/controller.py

```diff
@@ -152,14 +152,18 @@ def finalize_components(self, session: Session):  # noqa: B027
                 The current session containing the per-event configs
         """
 
-    def after_deploy(self, session: Session) -> bool:
+    def after_deploy(
```

Comment:
So this is a breaking change for consumers. @gabe-l-hart do you think that's fine or should we try to do some magic around backwards compatibility (like inspecting the functional to see how many args it takes)